app/decorators/service_provider_session_decorator.rb

```diff
@@ -119,6 +119,15 @@ def sp_alert_learn_more
     custom_alert? ? CUSTOM_SP_ALERTS.dig(sp_name, :learn_more) : 'https://login.gov/help/'
   end
 
+  # :reek:DuplicateMethodCall
+  def mfa_expiration_interval # rubocop:disable Metrics/AbcSize
+    aal_1_expiration = Figaro.env.remember_device_expiration_hours_aal_1.to_i.hours
+    aal_2_expiration = Figaro.env.remember_device_expiration_hours_aal_2.to_i.hours
+    return aal_2_expiration if sp.aal.present? && sp.aal > 1
+    return aal_2_expiration if sp.ial.present? && sp.ial > 1
```

Comment:
I recognize that this should be fine as written, and I'm not sure what the team's style is on this throughout the app -- but I know I've always appreciated using explicit parens for critical joint conditionals like this. For example: `if sp.ial.present? && (sp.ial > 1)`. Any thoughts on this nit?

Reply:
Yeah, the nil check here is no fun. I shuffled some things around to avoid it.